src/storage-blob-preview/azext_storage_blob_preview/tests/latest/test_storage_blob_live_scenarios.py

```diff
@@ -68,8 +74,19 @@ def verify_blob_upload_and_download(self, group, account, file_size_kb, blob_typ
         self.cmd('storage blob exists -n {} -c {}'.format(blob_name, container),
                  checks=JMESPathCheck('exists', False))
 
-        self.cmd('storage blob upload -c {} -f "{}" -n {} --type {}'
-                 .format(container, local_file, blob_name, blob_type))
+        def fix_block_blob_size(client, blob_type, length):
+            client._config.max_block_size = 4000 * 1024 * 1024
+            client._config.max_single_put_size = 5000 * 1024 * 1024
+
+        if fix_block_size:
+            import mock
+            with mock.patch('azext_storage_blob_preview.operations.blob._adjust_block_blob_size',
+                            side_effect=fix_block_blob_size):
+                self.cmd('storage blob upload -c {} -f "{}" -n {} --type {} --timeout 1200'
+                         .format(container, local_file, blob_name, blob_type))
```

Comment:
Mock function `_adjust_block_blob_size` here so that no matter how large the file is, `client._config` is always taking the largest value. In this way, jumbo blob can be tested.